app/helpers/application_helper.rb

```diff
@@ -1,6 +1,14 @@
 # frozen_string_literal: true
 
 module ApplicationHelper
+  # class methods
+  def self.batch_size
```

Comment:
This may be better suited to a `config/settings.yml` entry, and could benefit from a more specific name like `active_record_batch_size`, though I'm curious why we're redefining this at all

Reply:
there currently isn't a `config/settings.yml` so I would need to add gem 'config'

Reply:
Should adding `gem 'config'` be a separate PR ?

Reply:
Moved rubocop upgrade to https://github.com/department-of-veterans-affairs/gibct-data-service/pull/390
Adding `gem 'config'` to add `settings.yml`